test/e2e/specs/editor/editor__post-basic-flow.ts

```diff
@@ -86,7 +86,11 @@ describe( DataHelper.createSuiteTitle( 'Editor: Basic Post Flow' ), function ()
 
 	describe( 'Jetpack features', function () {
 		it( 'Open Jetpack settings', async function () {
-			await editorPage.openSettings( 'Jetpack' );
+			await editorPage.openEditorOptionsMenu();
+			const page = await editorPage.getEditorParent();
+
+			const button = await page.locator( 'button:has(span:has-text("Jetpack"))' );
```

Comment:
**suggestion (non-blocking)**: If you're open to it, I would recommend the React Testing Library style syntax here, like `getByRole( 'menuitemcheckbox', { name: "Jetpack" } );` We try to use that wherever possible! 😄

Reply:
Ahh, sure! I'll try it out :)